src/LiteCore/src/LiteCore.Shared/Interop/C4Replicator_defs.cs

```diff
@@ -74,7 +74,25 @@ internal unsafe struct C4Progress
         public ulong documentCount;
     }
 
-	internal unsafe struct C4Replicator
+    unsafe struct C4DocumentReplicatedStatus
```

Comment:
Don't edit this file by hand, it is auto-generated.  Where are these structs coming from?

Reply:
Where is `C4DocumentReplicatedStatus` defined?  This file should *only* be changed by hand in rare cases.  99% of the time it should be generated by the bindings generation script.